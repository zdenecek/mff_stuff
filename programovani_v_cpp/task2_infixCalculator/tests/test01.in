6
3.5
6/2
6/12
5.5-0.2
.
1 . 1
