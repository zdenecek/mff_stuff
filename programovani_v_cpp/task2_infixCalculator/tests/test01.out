3.5
3
0
5.3
invalid
invalid
